general/press-and-publications.rst

```diff
@@ -0,0 +1,153 @@
+- `video <http://www.youtube.com/watch?v=MY097Y2cPQ0&list=
+  PLPueLZei9c8_DEYgC5StOcR5bCAcQVfR8>`__
+- `abstract <https://archive.fosdem.org/2017/schedule/event/openwisp2>`__
+
+Do you really need to fork OpenWRT?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Presented at `OpenWRT Summit <http://openwrtsummit.org>`__ 2015 in Dublin
+
+- `video <https://www.youtube.com/watch?v=2uioGZuITbA>`__
+
+OpenWISP GARR Conference 2011
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interview for `GARR Conference <https://www.garr.it/en/>`_
+presented by Davide Guerri (in Italian)
+
+- `video <https://www.youtube.com/watch?v=4mxiupJNPKo>`__
+
+OpenWISP e Progetti WiFi Nazionali
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interview for `GARRTV <https://www.garr.tv>`_ by Davide Guerri
+(in Italian)
+
+- `video <https://www.youtube.com/watch?v=4AE7XSTPCT0>`__
+
+Blog Posts:
+-----------
+
+- `How Bottom-up Broadband will overcome the ‘last mile’ problem
+  <https://blog.p2pfoundation.net/how-bottom-up-broadband-will-overcome-
+  the-last-mile-problem/2013/07/23>`_
+- `netjsonconfig: convert NetJSON to OpenWRT UCI
+  <http://nemesisdesign.net/blog/coding/netjsonconfig-convert-netjson-to-
+  openwrt-uci/>`_
+- `Automate OpenWRT/LEDE firmware generation with Ansible
+  <http://nemesisdesign.net/blog/coding/automate-openwrt-lede-firmware
+  -generation-ansible/>`_
+- `django-x509: a reusable django app for PKI management
+  <http://nemesisdesign.net/blog/coding/django-x509-pki-pem/>`_
+- `Network Topology Visualizer: django-netjsongraph
+  <http://nemesisdesign.net/blog/coding/network-topology-visualizer-
+  django-netjsongraph/>`_
+- `Marco and Alessia for an increasingly open network
+  <https://uniamo.uniurb.it/openwisp/>`_ (in Italian)
+- `Fly with Uniurb and OpenWISP to the Google Summer of Code 2018
+  <https://uniamo.uniurb.it/google-summer-of-code-2018/>`_ (in Italian)
+- `Uniurb at the Google Summer of Code with OpenWISP2 and Marco
+  <https://uniamo.uniurb.it/uniurb-google-summer-of-code-openwisp2/>`_
+  (in Italian)
+- `Post by the Metrpolitan City of Rome
+  <http://www.cittametropolitanaroma.gov.it/homepage/elenco-siti-tematici
+  /wifimetropolitano/openwisp-la-soluzione-open-source-la-diffusione-
+  servizi-wifi/>`_ (in Italian)
+
+Research and publications:
+--------------------------
+
+- `Network Infrastructure as Commons
+  <https://github.com/openwisp/openwisp2-docs/blob/master/assets/
+  documents/netCommons.pdf>`_
+- `Bottom-up Broadband Initiatives in the Commons for Europe Project
+  <https://github.com/openwisp/openwisp2-docs/blob/master/assets/
+  documents/arxiv.org.pdf>`_
+- `Free Europe WiFi by Justel Pizarro
+  <https://github.com/openwisp/openwisp2-docs/blob/master/assets/
+  documents/JustelPizarro_2013.pdf>`_ (in Spanish)
+- `Bottom-up Broadband: Free Software Philosophy Applied to Networking
+  Initiatives
+  <https://github.com/openwisp/openwisp2-docs/blob/master/assets/documents
+  /dtic.upf.edu.pdf>`_ (only one mention but worth having it on the list)
```

Comment:
remove `(only one mention but worth having it on the list)`, that was a comment on my side and I didn't mean to add it to the page